shell_integration/windows/OCOverlays/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ add_library(OCOverlays MODULE
     OCOverlays.def
 )
 
+set_static_crt(OCContextMenu)
```

Comment:
s/OCContextMenu/OCOverlays/ !!

Reply:
Funnily enough it still works... so it indeed might be "inherited" from OCUtill...

Reply:
>  so it indeed might be "inherited" from OCUtill...

Because it's the same compilation unit then